docs/local-development.md

```diff
@@ -10,8 +10,12 @@ There are two ways to work on CircleCI docs locally: with Docker and with [Ruby]
 _(If you already cloned the project and forgot `--recurse-submodules`, run `git submodule update --init`)_
 1. Run `npm install` to fetch dependencies
 1. Run `npm run webpack-dev` to create needed js assets
+1. Add the following line to your `/etc/hosts` file:
+   ```bash
+   127.0.0.1 ui.circleci.com
+   ```
 1. Run `docker-compose up`
-1. The docs site will now be running on <http://localhost:4000/docs/>
+1. The docs site will now be running on <https://ui.circleci.com/docs/>. If the browser presents to you an HSTS Security Warning, you can safely bypass it as it is an expected outcome of running the Caddy Reverse Proxy in Docker.
```

Comment:
can we fix the numbering of lines and also maybe we can give warning of the Docker build time like so?
```suggestion
1. Run `npm install` to fetch dependencies
2. Run `npm run webpack-dev` to create needed js assets
13. Add the following line to your `/etc/hosts` file:
   ```bash
   127.0.0.1 ui.circleci.com
   ```
4. Run `docker-compose up` (Warning: This may take up to 10 minutes to build)
5. The docs site will now be running on <https://ui.circleci.com/docs/>. If the browser presents to you an HSTS Security Warning, you can safely bypass it as it is an expected outcome of running the Caddy Reverse Proxy in Docker.
```

Reply:
idk how to escape ``` in github suggestions

Reply:
Funny enough it still renders as `1 2 3 4 5 6` but good point!

Reply:
oh huh, is this some nerdy inside joke?

Reply:
> oh huh, is this some nerdy inside joke?
With GH markdown, prefacing `1.` at the beginning of lines automagically renders incremental lists ;)